### PR TITLE
congress: self auth to extend members permissions to do FunCall

### PR DESCRIPTION
**Pull request by robert-zaremba**
All members should be able to make a `FunctionCall` perm

### CODE REVIEW

File: congress/src/lib.rs [hunk 1]
```diff
@@ -546,6 +546,12 @@ impl Contract {
             emit_executed(prop_id);
         }
     }
+
+    pub fn add_fun_call_perm(&mut self) {
+        require!(env::predecessor_account_id() == env::current_account_id());
```
Comment (sczembor) [inline congress/src/lib.rs:551]: I think we need to check if admin no? I am not sure what we are checking here

File: congress/src/lib.rs [hunk 2]
```diff
@@ -546,6 +546,12 @@ impl Contract {
...
+        let mut m = self.members.get().unwrap();
+        m.1.push(PropPerm::FunctionCall);
```
Comment (sczembor) [inline congress/src/lib.rs:553]: isn't there a need to set members again? `self.members.set`

File: congress/src/lib.rs (hunk 1 above)
Comment (robert-zaremba) [reply congress/src/lib.rs:551]: There is no admin role, only contract itself. so will be similar to migration.

File: congress/src/lib.rs (hunk 2 above)
Comment (robert-zaremba) [reply congress/src/lib.rs:553]: yes, added

**Review by sczembor — APPROVED** [review]
Lgtm

**Pull request merged**